lib/Controller/RoomController.php

```diff
@@ -1491,4 +1491,15 @@ public function setMessageExpiration(int $seconds): DataResponse {
 
 		return new DataResponse();
 	}
+
+	#[NoAdminRequired]
+	#[RequireLoggedInModeratorParticipant]
+	public function setCanMentionEveryone(int $permission = Room::CAN_MENTION_EVERYONE_ALL): DataResponse {
+		try {
+			$this->roomService->setCanMentionEveryone($this->room, $permission);
```

Comment:
```suggestion
	public function setCanMentionEveryone(int $config): DataResponse {
		try {
			$this->roomService->setCanMentionEveryone($this->room, $config);
```

as per suggestion in the docs file

Reply:
But, isn't good to have a default value?

Reply:
No, clients simply must always send the value.
If there is a default value we could never change it, so clients can't rely on it anyway